roles/etcd/templates/etcd.j2

```diff
@@ -35,7 +35,15 @@ start)
     {% endmacro -%}
 
     {% macro init_cluster() -%}
-    export ETCD_INITIAL_CLUSTER_STATE=new
+
+    if [ ! -f /var/tmp/etcd.existing ]
+    then
+      export ETCD_INITIAL_CLUSTER_STATE=new
+      touch /var/tmp/etcd.existing
+    else
+      export ETCD_INITIAL_CLUSTER_STATE=existing
+    fi
+
```

Comment:
Would this impact an etcd upgrade? Let's see we are going from version 2.1 to 2.2. In that case, should we clean this up?